cirq-core/cirq/devices/device.py

```diff
@@ -253,3 +263,108 @@ def _from_json_dict_(cls, qubits, nx_graph, **kwargs):
         if nx_graph != '':
             graph_obj = nx.readwrite.json_graph.node_link_graph(nx_graph)
         return cls(qubits, graph_obj)
+
+
+@value.value_equality
+class GridDeviceMetadata(DeviceMetadata):
```

Comment:
There are already several other classes living in this file - thoughts on moving this one to a new file?

Reply:
Moved.